src/main/java/org/broadinstitute/hellbender/utils/GenotypeUtils.java

```diff
@@ -62,9 +62,19 @@ public static GenotypeCounts computeDiploidGenotypeCounts(final VariantContext v
                 continue;
             }
 
-            if (!g.hasLikelihoods() && g.isHomRef()) {
+            if (!g.hasLikelihoods()) {
+                if (!g.isHomRef() && !roundContributionFromEachGenotype) {
+                    throw new IllegalStateException("Genotype likelihoods cannot be determined from GQ for genotype: " + g +
+                        ".\nGenotypes with no PLs should have integer counts using roundContributionFromEachGenotype = true.");
```

Comment:
This error message seems confusingly worded to me.  Maybe something like "Non-hom-ref genotypes with no PLs are only allowed if roundContributionFromEachGenotype = true.